can/interfaces/seeedstudio/seeedstudio.py

```diff
@@ -94,11 +101,23 @@ def __init__(
                 "the serial module is not installed"
             )
 
+        if can_filters is None:
+            can_filters = [{"can_id": 0x00, "can_mask": 0x00}]
+
+        if len(can_filters) > 1:
```

Comment:
You could use the software fallback of python-can if the user passes multiple filters. Check the implementation in BusABC.